src/BlockchainData.js

```diff
@@ -0,0 +1,228 @@
+  		this.setState({ web3: web3 });
+  		const multisig = new web3.eth.Contract(MULTISIG_ABI, MULTISIG_ADDRESS);
+		this.setState({ multisig: multisig });
+		const multisigERC20Token = new web3.eth.Contract(MULTISIG_ERC20_ABI, MULTISIG_ERC20_ADDRESS);
+		this.setState({ multisigERC20Token: multisigERC20Token });
+		const tokenImperial = new web3.eth.Contract(ERC20_ABI, TOKENIMPERIAL_ADDRESS);
+		this.setState({ tokenImperial: tokenImperial });
+		const tokenDemocratic = new web3.eth.Contract(ERC20_ABI, TOKENDEMOCRATIC_ADDRESS);
+		this.setState({ tokenDemocratic: tokenDemocratic });
+		web3.eth.getBalance(this.state.etherAddress, (error, balance) => {
+			if (error) {
+				console.log(error)
+			} else {
+		        this.setState({ etherBalance: balance });
+	    		console.log('Account ether balance: ', this.state.etherBalance);
+	    	}
+			})
+		multisigERC20Token.methods.tokenBalances(this.state.etherAddress, TOKENIMPERIAL_SYMBOL).call({from: this.state.etherAddress}, (error, balance) => {
+			if (error) {
+		        console.log(error)
+		    } else {
+		        this.setState({ tokenImperialContractBalance: balance });
+		       	console.log('The imperial token contract balance of account is: ', this.state.tokenImperialContractBalance);
+		    }
+		})
+
+		multisigERC20Token.methods.tokenBalances(this.state.etherAddress, TOKENDEMOCRATIC_SYMBOL).call({from: this.state.etherAddress}, (error, balance) => {
+			if (error) {
+		        console.log(error)
+		    } else {
+		        this.setState({ tokenDemocraticContractBalance: balance });
+		       	console.log('The democratic token contract balance of account is: ', this.state.tokenDemocraticContractBalance);
+		    }
+		})
+
+		multisig.methods.balances(this.state.etherAddress).call({from: this.state.etherAddress}, (error, balance) => {
+		    if (error) {
+		        console.log(error)
+		    } else {
+		        this.setState({ contractBalance: balance });
+		       	console.log('The contract balance of account is: ', this.state.contractBalance);
+		    }
+		})
+
+		tokenImperial.methods.balanceOf(this.state.etherAddress).call({from: this.state.etherAddress}, (error, balance) => {
+		    if (error) {
+		        console.log(error)
+		    } else {
+		        this.setState({ tokenImperialBalance: balance });
+		       	console.log('The token Imperial balance of account is: ', this.state.tokenImperialBalance);
+		    }
+		})
+
+		tokenDemocratic.methods.balanceOf(this.state.etherAddress).call({from: this.state.etherAddress}, (error, balance) => {
+		    if (error) {
+		        console.log(error)
+		    } else {
+		        this.setState({ tokenDemocraticBalance: balance });
+		       	console.log('The token Democratic balance of account is: ', this.state.tokenDemocraticBalance);
+		    }
+		})
+
+		//this.interval = setInterval(this.updateBalances, 15000);
+		//this.interval = updateERCBalances(this.updateERCBalances, 15000);
+  	}
+
+  	componentWillUnmount() {
```

Comment:
do same code style, 
some functions you write as `componentWillUnmount() {...}`
and some as`  updateBalances = () => {...}`